tests/compile/callingconvention.srctree

```diff
@@ -35,10 +35,19 @@ p2 = f2
 p3 = f3
 p4 = f4
 
+
+######## callingconvention.h ########
+
+#define DLL_EXPORT
+#include "external_callingconvention.h"
+
+
 ######## external_callingconvention.h ########
 
 #ifndef DL_IMPORT
   #define DL_IMPORT(t) t
+#elif defined(DLL_EXPORT)
+  #define DL_IMPORT(t) DL_EXPORT(t)
```

Comment:
Ok, then what about `USE_DL_EXPORT` or `EXPORT_SYMBOLS` ? Or anything else that makes it clear what the purpose is.